liquidctl/driver/smart_device.py

```diff
@@ -111,6 +111,8 @@
 SPDX-License-Identifier: GPL-3.0-or-later
 """
 
+import numpy as np
```

Comment:
I don't see this being used?

Reply:
whoops, true. I have removed numpy (and time, which was apparently also unused)